### PR TITLE
Headsign best practices

### PR DESCRIPTION
**Pull request by antrim**
This is one of 5 PRs that incorporates a recommendation from the GTFS best practices.

This PR adds explains that stop_headsign does not carry down.

Related pull requests:
- [Add best practice: stop_id is not rider facing](https://github.com/google/transit/pull/169)
- [Add best practice: In seat transfer specifics](https://github.com/google/transit/pull/168)
- [Add best practices: Headsign and route name examples](https://github.com/google/transit/pull/166)
- [Arrival and departure time best practices](https://github.com/google/transit/pull/165)

### CODE REVIEW

File: gtfs/spec/en/reference.md [hunk 1]
```diff
@@ -200,9 +200,9 @@ File: **Required**
...
+|  `stop_headsign` | Text | Optional | Text that appears on signage identifying the trip's destination to riders. This field overrides the default `trips.trip_headsign` when the headsign changes between stops. If the headsign is displayed for an entire trip, use `trips.trip_headsign` instead. <br><br> `stop_headsign` values do not “carry down” to subsequent stops, and therefore values must be repeated if the stop headsign remains the same. |
+|  `pickup_type` | Enum | Optional | Indicates drop off method. Valid options are:<br><br>`0` or empty - Regularly scheduled pickup. <br>`1` - No pickup available.<br>`2` - Must phone agency to arrange pickup.<br>`3` - Must coordinate with driver to arrange pickup. |
+|  `drop_off_type` | Enum | Optional | Indicates pick up method. Valid options are:<br><br>`0` or empty - Regularly scheduled drop off.<br>`1` - No drop off available.<br>`2` - Must phone agency to arrange drop off.<br>`3` - Must coordinate with driver to arrange drop off. |
```
Comment (paulswartz) [inline gtfs/spec/en/reference.md]: ❓

**Comment by HeidiMG** [comment]
Typo @paulswartz pointed out has been fixed.

**Comment by antrim** [comment]
This pull request has been open for more than one week, so per the Official Process I'm calling for a vote.

Vote will be closed on Tuesday, June 25th at 23:59:59 UTC.

**Comment by abyrd** [comment]
On first reading this change it was not clear to me what "carry down" meant, so you may want to rephrase this. Also, it could be helpful to use more precise terms than "subsequent stops". Suggestion: 

A `stop_headsign` value specified for one `stop_time` does not apply to subsequent `stop_time`s in the same trip. If you want to override the `trip_headsign` for multiple `stop_time`s in the same trip, the `stop_headsign` value must be repeated in each `stop_time` row.

**Comment by antrim** [comment]
@abyrd Thanks for the suggested improvement. I committed that change to the pull request. I'm calling this the start of the voting period, and extending the end to Wednesday, June 26th at 23:59:59 UTC.

File: gtfs/spec/en/reference.md [hunk 2]
```diff
@@ -200,8 +200,8 @@ File: **Required**
...
-|  `pickup_type` | Enum | Optional | Indicates pickup method. Valid options are:<br><br>`0` or empty - Regularly scheduled pickup. <br>`1` - No pickup available.<br>`2` - Must phone agency to arrange pickup.<br>`3` - Must coordinate with driver to arrange pickup. |
+|  `stop_headsign` | Text | Optional | Text that appears on signage identifying the trip's destination to riders. This field overrides the default `trips.trip_headsign` when the headsign changes between stops. If the headsign is displayed for an entire trip, use `trips.trip_headsign` instead. <br><br>  A `stop_headsign` value specified for one `stop_time` does not apply to subsequent `stop_time`s in the same trip. If you want to override the `trip_headsign` for multiple `stop_time`s in the same trip, the `stop_headsign` value must be repeated in each `stop_time` row. |
+|  `pickup_type` | Enum | Optional | Indicates pick up method. Valid options are:<br><br>`0` or empty - Regularly scheduled pickup. <br>`1` - No pickup available.<br>`2` - Must phone agency to arrange pickup.<br>`3` - Must coordinate with driver to arrange pickup. |
```
Comment (TomGoBravo) [inline gtfs/spec/en/reference.md]: Why change only one instance of "pickup" to "pick up"?

**Comment by TomGoBravo** [comment]
+1

**Comment by mgilligan** [comment]
+1

**Comment by mgilligan** [comment]
+1, I've been meaning to fix TriMet's GTFS to properly use `trip_headsign` and remove our `stop_times.txt` bloat. Thanks for the reminder/motivation.

File: gtfs/spec/en/reference.md (hunk 2 above)
Comment (HeidiMG) [reply gtfs/spec/en/reference.md]: Sorry - was focused on correcting the error and didn't catch the inconsistency. Fixed now.

**Comment by paulswartz** [comment]
+1

**Comment by ibi-group-team** [comment]
+1

**Comment by harringtonp** [comment]
+1

**Comment by thomastrillium** [comment]
+1

**Comment by antrim** [comment]
6 votes are in favor, including at least one consumer (@harringtonp) and producer. 0 votes against. The vote passes.

**Pull request merged**